skills/mindroom-docs/references/page__chat-commands__index.md

```
@@ -118,6 +118,15 @@ Add a phrase such as `with only the last 5 messages of context` to cap each sche
 !schedule Daily at 9am, @research summarize AI news with only the last 5 messages
 ```
 
+Add `silently` or `quietly` when a scheduled check should post only findings, failures, or messages explicitly sent by tools.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)